app/views/documents/history.html.erb

```diff
@@ -14,63 +14,7 @@
           <h3 class="govuk-heading-m"><%= t "documents.history.edition_header", number: entry.edition.number.ordinalize %></h3>
           <% displayed_edition_number = entry.edition.number %>
         <% end %>
-
-        <% if entry.imported_from_whitehall? %>
-          <%= render "govuk_publishing_components/components/notice", {
-            title: "#{t "documents.history.entry_types.#{entry.entry_type}"}",
-            description_text: format_date(entry.created_at)
-          } %>
-        <% else %>
-
-          <h4 class="govuk-heading-s govuk-!-margin-bottom-1">
-            <%= t "documents.history.entry_types.#{entry.entry_type}" %>
-          </h4>
-
-          <div class="app-timeline-entry__dateline">
-            <%= format_date_at_time(entry.created_at) %>
-            <% if entry.created_by %>
-              by <%= name_or_fallback(entry.created_by) %>
-            <% end %>
-          </div>
-
-          <% if entry.internal_note? && entry.details %>
-            <%= simple_format(escape_and_link(entry.details.body)) %>
-          <% end %>
-
-          <% if (entry.withdrawn? || entry.withdrawn_updated?) && entry.details %>
-            <%= entry.details.public_explanation %>
-          <% end %>
-
-          <% if entry.backdated? %>
-            <% date = format_date(entry.revision.backdated_to) %>
-            <%= t "documents.history.entry_content.backdated", date: date %>
-          <% end %>
-
-          <% if entry.removed? && entry.details %>
-            <% removal = entry.details %>
-            <% if removal.explanatory_note.present? %>
-              <p><%= removal.explanatory_note %></p>
-            <% end %>
-
-            <% if removal.alternative_path.present? %>
-              <% if removal.redirect? %>
-                <p>
-                  Redirected to
-                  <%= link_to(Plek.new.website_root + removal.alternative_path,
-                              nil,
-                              class: "govuk-link") %>
-                </p>
-              <% else %>
-                <p>
-                  Alternative path
-                  <%= link_to(Plek.new.website_root + removal.alternative_path,
-                              nil,
-                              class: "govuk-link") %>
-                </p>
-              <% end %>
-            <% end %>
-          <% end %>
-        <% end %>
+        <%= render partial: "documents/timeline_entry", locals: {entry: entry} %>
```

Comment:
Minor, convention would be to have spaces within the braces. For example: `{ entry: entry }`